Source/WebCore/platform/graphics/gstreamer/ImageGStreamerSkia.cpp

```diff
@@ -85,7 +85,7 @@ ImageGStreamer::ImageGStreamer(GRefPtr<GstSample>&& sample)
     }
 
     auto imageInfo = SkImageInfo::Make(videoFrame->width(), videoFrame->height(), colorType, alphaType);
-    SkPixmap pixmap(imageInfo, videoFrame->planeData(0), videoFrame->planeStride(0));
+    SkPixmap pixmap(imageInfo, videoFrame->planeData(0), videoFrame->planeStride(0).value_or(-1));
```

Comment:
You can use value() here, we return early if video frame is invalid. I still think we should assert instead of handling mapped frame in invalid state

Reply:
Do you mean asserting when creating the GstMappedFrame?

Reply:
No, I mean stop handling m_isValid in every GstMappedFrame accessor and assert instead.